server/android/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <uses-feature android:name="android.hardware.usb.host" />
```

Comment:
I'm not sure if specifying this means that the app *requires* this API to function. On the docs, they say that this indicates that the app *uses* the APIs, but not what the consequence of specifying this does.

Reply:
it means [it needs it for it to run](https://developer.android.com/guide/topics/manifest/manifest-intro#uses-feature)

Reply:
> For example, if your app can't achieve basic functionality on a device without a compass sensor ...

So this probably shouldn't be specified then, got it

Reply:
you can make it optional by adding a `android:required="false"` inside it and probably checking for the feature being available via [`hasSystemFeature()`](https://developer.android.com/reference/android/content/pm/PackageManager#FEATURE_USB_HOST)